packages/compass-aggregations/src/modules/create-view/index.spec.js

```diff
@@ -40,8 +40,8 @@ describe('create view module', function () {
           dataService: { dataService: 'ds' },
         });
 
-        before(function () {
-          createView()(dispatchSpy, getState);
+        before(async function () {
+          await createView()(dispatchSpy, getState);
```

Comment:
just a question, is this `await` necessary now? or it would be the same if we don't await?

Reply:
It's just to make sure that the promise returned from the `createView` thunk action is fully went through and all dispatches happened, technically taking into account that the mock is passing a string instead of a data service that is not really required, but as the promise is returned I prefer to await it

(also I just noticed that the only test in this block is skipped anyway, so this affects even less things that I initially thought)